test/integration/admin/api/backend_apis/metrics_controller_test.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Admin::API::BackendApis::MetricsControllerTest < ActionDispatch::IntegrationTest
+  def setup
+    @tenant = FactoryBot.create(:provider_account)
+    host! @tenant.admin_domain
+    @access_token_value = FactoryBot.create(:access_token, owner: @tenant.admin_users.first!, scopes: %w[account_management], permission: 'rw').value
+    @backend_api = FactoryBot.create(:backend_api, account: @tenant)
+  end
+
+  attr_reader :backend_api, :access_token_value, :tenant
+
+  test 'index' do
+    FactoryBot.create(:metric, owner: backend_api, parent: backend_api.metrics.hits)
```

Comment:
Does `FactoryBot` create an associated `Service` with the metric anyway? Not sure if we haven't to pass `service_id : nil` 🤔

Reply:
Good point 👍 I will check 👍

Reply:
Checked: it is true but it only happens for the parent metrics, not children.
I will fix it well in https://github.com/3scale/porta/pull/1204 but now I've just taken the fastest approach: `service_id: nil`